src/main/java/dev/pace/staffchat/Main.java

```diff
@@ -47,7 +46,7 @@ public void onEnable() {
         getCommand("sc").setExecutor(new StaffChat());
         getCommand("sctoggle").setExecutor(new StaffChatToggle());
         getCommand("adminchat").setExecutor(new AdminChat());
-        getCommand("asc").setExecutor(new AdminChat());
+        if (config.getBoolean("adminchat-enabled")) getCommand("asc").setExecutor(new AdminChat());
```

Comment:
Since there's a check here, how about the other 2 commands on the bottom? You could probably just copy and paste the conditional for the other 2